db/fixtures/products.yml

```
@@ -124,3 +124,21 @@ product63:
   practice: practice6
   user: kimura
   body: プラクティス完了メッセージ未表示の提出物です。
+
+<% (0..3).each do |i| %>
+product<%= 64 + i %>:
+  practice: practice<%= 4 + i %>
+  user: hatsuno
+  body: <%= i+1 %>日前の提出物です。
+  published_at: <%= (now - (60 * 60 * 24 * (i+1))).to_s(:db) %>
+  created_at: <%= (now - (60 * 60 * 24 * (i+1))).to_s(:db) %>
```

現在時刻からの計算は [ago](https://api.rubyonrails.org/classes/ActiveSupport/Duration.html#method-i-ago) や [since](https://api.rubyonrails.org/classes/ActiveSupport/Duration.html#method-i-iso8601) を使う方が短く書けます。
あと、 `to_s(:db)` はRails 7から警告が出るので `to_formatted_s(:db)` の方が良いかも。
（Rails 7からは `to_fs(:db)` で書けるけど、6.1にはメソッドが存在しないので）
参考: https://github.com/rails/rails/blob/v7.0.2.3/activesupport/lib/active_support/time_with_zone.rb#L214-L216

```suggestion
  created_at: <%= (i+1).ago.to_formatted_s(:db) %>
```